src/nanoemoji/write_font.py

```diff
@@ -363,6 +364,22 @@ def _draw_glyph_extents(
     return glyph
 
 
+def _draw_notdef(config: FontConfig, ufo: ufoLib2.Font):
+    # A StubGlyph named .notdef provides a nice drawing of a notdef
+    notdefArtist = StubGlyph(
+        ".notdef",
+        config.width,
+        config.upem,
+        config.ascender,
+        config.descender,
+    )
+
+    # UFO doesn't like just sticking StubGlyph directly in place
```

Comment:
@anthrotype ufo seemed angry when I tried to simply assign stubglyph as the definition of .notdef so I drew it on instead.

Reply:
All in all, I think what you did is actually better than trying to stick in an alien glyph type